apps/style/JsDebuggerUi.scss

```diff
@@ -29,6 +29,13 @@
   padding: 3px 6px;
   font-size: 14px;
   min-width: 120px;
+
+  &:disabled {
+    cursor: not-allowed;
+    &:active {
```

Comment:
why was this change to `&:active` needed? Can we do it without using `!important`?

Reply:
The problem is that we have !importnat styles in common.scss (apps/style/common.scss , line 628) that makes disabled active button to have a black border. We can leave this border for active status if it's ok, but I like current option more.

1. before updates 
![Screenshot from 2023-01-04 20-35-15](https://user-images.githubusercontent.com/22244040/210626890-1c2ded7b-429a-4d0b-ac32-13686da7c3c5.png)
2. after updates 
![Screenshot from 2023-01-04 20-34-44](https://user-images.githubusercontent.com/22244040/210626932-8545f41d-8bfa-4224-a10d-ce89d0c57590.png)

Reply:
I'll defer to @moneppo for if the black border is okay - if we want to remove it, can you add a comment explaining why the !important is needed?

Reply:
to rewrite the !important we have at (apps/style/common.scss , line 628) that makes disabled active button to have a black border.